test_queue_job/tests/test_job_auto_delay.py

```diff
@@ -32,16 +32,7 @@ def test_auto_delay_inside_job(self):
 
     def test_auto_delay_force_sync(self):
         """method forced to run synchronously"""
-        with self.assertLogs(level="WARNING") as log_catcher:
-            result = (
-                self.env["test.queue.job"]
-                .with_context(_job_force_sync=True)
-                .delay_me(1, kwarg=2)
-            )
```

Comment:
why aren't we keeping this `_job_force_sync` context key in the `test_auto_delay_force_sync` test method ?

Reply:
Hi @mmequignon ,
It was announced as no longer available from v17 so I removed it. ([Here](https://github.com/OCA/queue/pull/692/commits/8c92c88e3dcc50974974545f44555a24eb6a9ed2#diff-6e33951ef77e0428f3edcd938952cf20d30ca84e703406c08685f3d70d68da86L27-L37))